rules.js

```diff
@@ -100,7 +101,10 @@ module.exports = function(styles) {
         state.withinText = true;
         return React.createElement(Text, {
           key: state.key,
-          style: styles.autolink
+          style: styles.autolink,
+          onPress: function () {
+            Linking.openURL(node.content[0].content);
```

Comment:
I think this is grabbing the output text (what's in content). The `link` node should have a `target` element, and I'd recommend the following here:

```javascript
Linking.openURL(SimpleMarkdown.sanitizeUrl(node.target));
```

which is based on https://github.com/Khan/simple-markdown/blob/master/simple-markdown.js#L1071